### ISSUE TITLE
[Suggestion] Add proper and native Windows support

### ISSUE THREAD

**Issue by SaurikSI** [Al4ise/Azule#104]
I have been experimenting with Azule on SatellaJailed recently, and I realized that proper Windows support is very much needed, right now the main issue is with install_name_tool, which doesn't exist for Linux and Windows, there is a Linux port here on GitHub but it doesn't work for Azule's purpose.

A native support would be even better, because WSL sometimes conflicts with IPA patching, Paisseon (Satella creator) and I have exchanged ideas about issues with their tweak's injection, which sometimes ended up being an Azule issue, so implementing this would help a lot. Paisseon has suggested implementing hex patches to replace install_name_tool, I honestly don't know if that would be stable, but it's worth considering, please [take a look](https://github.com/Paisseon/SatellaJailed/issues/3)

Making a native port is definitely plausible, but there could be another way that would also help Windows users, but I don't know if it's possible: Jailed iOS support. There are Alpine terminal apps for jailed iOS, and apps can read-write importing files, so I don't see exactly what limitations could prevent this, so I just leave my idea.

Thanks for developing Azule, your work helps the community a lot, I hope we can make one of these ideas possible.

**Comment by Al4ise** [comment]
1. Native windows support is.. I mean I have no idea how to do it. Many of the tools needed are macos only. There are a few linux ports available which I am using for the linux support, but I've never found a toolchain like that for windows
2. I've looked into jailed ios support, but iSH is just not stable enough for azule's purposes. And I don't know of any alternatives, sadly. Though there may or may not be a project that would make azule accessible from the web (hint hint), no eta cause I'm not the developer there

**Comment by SaurikSI** [comment]
> 1. Native windows support is.. I mean I have no idea how to do it. Many of the tools needed are macos only. There are a few linux ports available which I am using for the linux support, but I've never found a toolchain like that for windows
> 2. I've looked into jailed ios support, but iSH is just not stable enough for azule's purposes. And I don't know of any alternatives, sadly. Though there may or may not be a project that would make azule accessible from the web (hint hint), no eta cause I'm not the developer there

1. Yeah, unfortunately I really don't know programming, so I can't help with this. If you're comfortable with this, maybe you should  involve someone experienced with programming on Windows and who's interested on this project, the tweaking community is very big so I'm sure there must be someone.

2. I'm not sure how different it is, but there are other terminal apps like a-Shell. That web project would be awesome, I suggested the guys behind UDIDRegistrations.com and Signulous to add online Azule injecting support, they said they will think about it, maybe you could contact them and see what they think.

Azule as a project is very useful, but if signing services implement it, this could be huge.

**Comment by Al4ise** [comment]
i mean scarlet has a tweak injector? SignTools has a tweak injector. azule is not even close to the only thing in the game. but yeah about the windows support, I can't promise anything :​/ sorry

**Comment by SaurikSI** [comment]
> i mean scarlet has a tweak injector? SignTools has a tweak injector. azule is not even close to the only thing in the game. but yeah about the windows support, I can't promise anything :​/ sorry

Yes, but Azule has the best compatibility and reliability, less crashes on my experience.
Don't worry about the Windows support, I'm not looking for a promise, just wanted to let you know why I think it is important, but if you or another dev makes it, I will gladly help testing.
